### ISSUE TITLE
Circle is not displaying correctly

### ISSUE THREAD

**Issue by UHemarajU** [OrbJapan/ResizableMKCircleOverlay#6]
Hi Orb Japan,

Thanks for the example. I working fine in iOS 9. But when is run this app in iOS  10 some part of the the circle is cutting. Please help me to resolve this.

Thnak you.

**Comment by MDBox** [comment]
Hello,

It has been awhile since I've looked at this but I think I found a solution for IOS 10.  When you make the MKCircle object make sure you set the radius to the maximum allowed value for the circle. 

In the example we set a maximum value to 2.0 KM.  So after I updated this line of code it worked again in IOS 10.  

`circle = [MKCircle circleWithCenterCoordinate:droppedAt radius:2000.0];`

I think the problem was IOS 10 changed how the bounding box around the circle is set.  It wasn't making the drawing area large enough so the circle was being clipped to this smaller region.  I could be wrong but that is what I'm thinking.  I'll make a pull request to fix the example and ask the guys at 'Orb Japan' to update the repo.
